<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_entities_form</class>
 <widget class="QWidget" name="db_entities_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QToolButton" name="tbEditEntity">
     <property name="toolTip">
      <string>Edit record</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="tbAddEntity">
     <property name="toolTip">
      <string>Add resord</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>360</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="tbDelEntity">
     <property name="toolTip">
      <string>Delete record</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tbRefresh">
     <property name="toolTip">
      <string>refreshModel</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QTreeView" name="tvViewEntities"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
